app/backend/aquifers/migrations/0026_auto_20191216_2135.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 2.2.8 on 2019-12-16 21:35
+
+from django.db import migrations
+
+
+insertGWSWInteractionsSection = """
+    INSERT INTO aquifer_resource_section_code (
+        aquifer_resource_section_code,
+        create_user,
+        create_date,
+        update_user,
+        update_date,
+        effective_date,
+        expiry_date,
+        name,
+        description
+    )
+    VALUES (
+        'G',
+        'DATALOAD_USER',
+        '2020-01-01T08:00:00Z',
+        'DATALOAD_USER',
+        '2020-01-01T08:00:00Z',
+        '1970-01-01T08:00:00Z',
+        '9999-12-31T23:59:59Z',
+        'Groundwater Surface Water Interactions',
```

Comment:
I think this should be Groundwater surface water interactions, since it's not a title.